src/fswAlgorithms/orbitControl/lambertSurfaceRelativeVelocity/_UnitTest/test_lambertSurfaceRelativeVelocity.py

```diff
@@ -0,0 +1,170 @@
+#
+#  ISC License
+#
+#  Copyright (c) 2023, Autonomous Vehicle Systems Lab, University of Colorado Boulder
+#
+#  Permission to use, copy, modify, and/or distribute this software for any
+#  purpose with or without fee is hereby granted, provided that the above
+#  copyright notice and this permission notice appear in all copies.
+#
+#  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+#  WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+#  MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+#  ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+#  WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+#  ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+#  OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+#
+#
+import itertools
+
+import numpy as np
+import pytest
+from Basilisk.architecture import messaging
+from Basilisk.fswAlgorithms import lambertSurfaceRelativeVelocity
+from Basilisk.utilities import RigidBodyKinematics
+from Basilisk.utilities import SimulationBaseClass
+from Basilisk.utilities import macros
+from Basilisk.utilities import orbitalMotion
+
+# parameters
+relativeVelocities = np.array([[0., 0., 0.], [100., -300., 200.]])
+time_maneuver = [1.e3, 2.e3]
+trueAnomalies = [0., 70.]
+rotAngles = [0., 230.]
+angularVelocities = np.array([[0., 0., 0.], [1., -3., 2.]])
+
+paramArray = [relativeVelocities, time_maneuver, trueAnomalies, rotAngles, angularVelocities]
+# create list with all combinations of parameters
+paramList = list(itertools.product(*paramArray))
+
+@pytest.mark.parametrize("accuracy", [1e-4])
+@pytest.mark.parametrize("p1_vr, p2_tm, p3_f, p4_rot, p5_omega", paramList)
+
+def test_lambertSurfaceRelativeVelocity(show_plots, p1_vr, p2_tm, p3_f, p4_rot, p5_omega, accuracy):
+    r"""
+    **Validation Test Description**
+
+    This test checks if the Lambert Surface Relative Velocity module works correctly for different relative velocity
+    vectors, maneuver times, true anomalies, planet rotation angles, and planet angular velocity vectors.
+
+    **Test Parameters**
+
+    Args:
+        :param show_plots: flag if plots should be shown
```

Comment:
Your function arguments are not showing up properly.  I think you need to remove the `:param`.  Here is an example that works

```
Args:
        show_plots (bool): Determines if the script should display plots
        missionType (str):

            ===========  ==================================
            String       Definition
            ===========  ==================================
            'dscovr'     Simulates the NASA DSCOVR mission
            'marsOrbit'  Simulates an orbit about Mars
            ===========  ==================================

        saveVizardFile (bool): Flag to save off the Vizard data file
```

Please remove `:param` from all `Args:` function doc-strings.

Reply:
Done.